### ISSUE TITLE
Manual delete of backups

### ISSUE THREAD

**Issue by fedpettinella** [mail-in-a-box/mailinabox#1683]
Hello,

I'm preparing to move my MIAB from v0.30 to the latest, however I can't create the backup because it runs out of space. There are about 15.5 GB of backups on the device (30 GB total disk) - can I manually delete them before running the backup.py command? If I clear them I should be able to do a full backup, however I'm worried that a manual deletion will break the backup process and take my data with it. What's your advice in this situation? Can I just delete the backups/cache folder and run backup.py again?

Thank you

**Comment by alento-group** [comment]
Take a look at the date in the filenames in your encrypted backups directory ... if that backup is current, then I would say yes, no problem.  If they are older backups, I would be a bit concerned.

Even if the worse case scenario were to happen and you are not able to create a new backup, your data is still safe.  If this were to happen you'd need to rsync the data from the old box to the new one, which could be problematic if you are trying to preserve the IP address ... but let's only cross that bridge if it becomes necessary.

**Comment by fedpettinella** [comment]
Thanks for the tip @alento-group . I tried `rm -f /home/user-data/backup/cache/xxxxxxxxx/*` and then attempted to run `sudo /root/mailinabox/management/backup.py`. Unfortunately it just came back with the timestamp in the CLI. When I went to the backups page in the web UI then I got the same timestamp:
![image](https://user-images.githubusercontent.com/17450703/69500062-ac5d7500-0eef-11ea-9bc2-e84563750705.png)


Luckily I downloaded the whole folder yesterday so right now I'm re-uploading them to the server, then I can try something else. I noticed that the option in my /encrypted/ folder is about a month old. Not terrible, considering this is not a high-velocity server... But also not great.

Any other ideas? :)

**Comment by alento-group** [comment]
Make sure that you have your secret key file saved some place safe ... then delete the entire contents of the /home/user-data/backup/encrypted/ directory.
After that, rerun `/root/mailinabox/management/backup.py`

This should create a new backup.

**Comment by alento-group** [comment]
> I noticed that the option in my /encrypted/ folder is about a month old.

Is that the oldest or newest date?  There will be anywhere from 1-30 files depending how large your backups are.

**Comment by fedpettinella** [comment]
The newest date was around 25th of October yes. Seems that the backups haven't gone through successfully for a month! 

But that's okay as your tip worked. I deleted all files within the `/encrypted/` folder and then re-ran the `backup.py` command. The command took a few minutes to complete but then I went from 200 MB free (pre-deletion) to 19 GB free (post-deletion) to now about 14 GB free (after running backup). I deleted a few user mailboxes and some big folders in NextCloud so the actual size decreased quite dramatically.

Now I'm in a place that I can go restore my box onto a v0.4x box! Thank you @alento-group!
